error.go

```diff
@@ -2,38 +2,42 @@
 // and efficient error handling.
 package microerror
 
-import (
-	"fmt"
-
-	"github.com/juju/errgo"
-)
-
 var (
-	// MaskAny is a simple error masker. Masked errors act as tracers within the
-	// source code. Inspecting an masked error shows where the error was passed
-	// through within the code base. This is gold for debugging and bug huntin.
-	MaskAny = errgo.MaskFunc(errgo.Any)
+	Default = NewErrgoHandler(ErrgoHandlerConfig{
+		CallDepth: 1,
```

Comment:
I do not get why `Default`/`InternalHandler` have to be public. It seems to just be some implementation we use internally. Fine. It is not supposed to be used externally, so it should be private.